pkg/ticker/ticker.go

```diff
@@ -138,9 +136,7 @@ func (r *ResyncTicker) startTicker(tickIterval time.Duration) {
 			return
 
 		case <-ticker.C:
-			r.msgBroker.GetQueue().AddRateLimited(events.PubSubMessage{
-				Kind: announcements.ProxyUpdate,
-			})
```

Comment:
The main reason is that it guarantees that events have a kubetnetes object type. 

@shashankram did you mention we could even just remove the ticker code completely?

Reply:
Ticker is a hack and/or diagnostic mechanism to force events through the system if something gets missed and to help the system recover in prod if there's a bug. It would make sense to route it through the workqueue (without rate limiting) to get additional stats and logging.

Could you ensure this goes through the centralized eventing framework so we get the necessary metrics because that's pretty important when Ticker is turned on.

Reply:
done!